src/VisualStudio/DevKit/Impl/Microsoft.VisualStudio.LanguageServices.DevKit.csproj

```diff
@@ -48,15 +48,16 @@
   
   <!--
   Add these additional runtime dependencies to our NuGet package.
+  Ensure they are copied to the build output directory for ease of validating local changes in the C# extension
   -->
-  <Target Name="CollectPackInputs" DependsOnTargets="Build" BeforeTargets="Pack" Returns="@(Content)">
+  <Target Name="CollectPackInputs" AfterTargets="ResolveProjectReferences" Returns="@(Content)">
     <ItemGroup>
-      <Content Include="$(PkgMicrosoft_VisualStudio_Telemetry)\lib\netstandard2.0\Microsoft.VisualStudio.Telemetry.dll" Pack="true" PackagePath="content" />
-      <Content Include="$(PkgMicrosoft_VisualStudio_RemoteControl)\lib\netstandard2.0\Microsoft.VisualStudio.RemoteControl.dll" Pack="true" PackagePath="content" />
-      <Content Include="$(PkgMicrosoft_VisualStudio_Utilities_Internal)\lib\netstandard2.0\Microsoft.VisualStudio.Utilities.Internal.dll" Pack="true" PackagePath="content" />
-      <Content Include="$(PkgSystem_Configuration_ConfigurationManager)\lib\netstandard2.0\System.Configuration.ConfigurationManager.dll" Pack="true" PackagePath="content" />
-      <Content Include="$(PkgMicrosoft_VisualStudio_Debugger_Contracts)\lib\netstandard2.0\Microsoft.VisualStudio.Debugger.Contracts.dll" Pack="true" PackagePath="content" />
-      <Content Include="$(TargetPath)" Pack="true" PackagePath="content" />
+      <Content Include="$(PkgMicrosoft_VisualStudio_Telemetry)\lib\netstandard2.0\Microsoft.VisualStudio.Telemetry.dll" Pack="true" PackagePath="content" CopyToOutputDirectory="PreserveNewest" />
```

Comment:
I wonder if we should move these Content items out of the target (directly to the project).
Then the target can just return all Content items that have "Pack=true". Or perhaps NuGet would include them automatically and we might not need the target at all?

Reply:
I can try that but I'm not sure if the PkgXYZ variables are defined when the content items get evaluated.  Will check.

Reply:
Ok - that mostly worked.  I don't need the target to build and pack the devkit project.  But I do need a target to access the content in the LanguageServer.UnitTests project (just calling the build target doesn't get the all the copied items).